build/build.sh

```diff
@@ -101,6 +101,12 @@ if [[ "$AOT_BUILD_NEEDED" -eq 1 ]]; then
   popd
 fi
 
+# create an empty F# app so the fsharp core library is downloaded from nuget
+pushd /tmp
+"${DIR}/dotnet.sh" new console -lang F# -o app
+"${DIR}/dotnet.sh" publish app
+popd
+
```

Comment:
I think this is no longer needed as the F# core library already present in the sdk (it just need to be extracted to the file system while restoring), and we have fixed the issue where it failed to extract the package due to readonly fs.

Reply:
I agree, this is working again, also in older versions. (older versions was caused by a bug in our sandboxing)